src/main/java/demon/Commands.java

```diff
@@ -0,0 +1,285 @@
+package demon;
+
+import java.io.IOException;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * This class serves as a central point for managing and executing a variety of user commands.
+ * It encapsulates the logic for each command as individual methods, allowing for easy execution
+ * based on user input. Commands such as 'list', 'unmark', 'mark', 'delete', and others are supported, each
+ * corresponding to a specific method within this class.
+ */
+public class Commands {
```

Comment:
Hi Cole! Small point here, but maybe you can consider naming it just 'Command' as the plural form make it seem like it is a collection of commands? 🤔

Reply:
Second this!